insightconnect_plugin_runtime/api/endpoints.py

```diff
@@ -568,29 +568,37 @@ def connection_test():
                   description: Internal server error
             """
             status_code = 200
-            message = None
+            output = None
+
+            input_message = request.get_json(force=True)
+            self.logger.debug("Request input: %s", input_message)
+            Endpoints.validate_action_trigger_task_empty_input(input_message)
+
             try:
-                if hasattr(self.plugin.connection, "test"):
-                    response = self.plugin.connection.test()
-                    if response is None:
-                        status_code = 204
-                        message = "The server successfully processed the request and is not returning any content"
-                    else:
-                        message = response
-                else:
+                output = self.plugin.handle_step(input_message, is_debug=self.debug, is_test=True)
+                if output.get("body").get("output") is None:
```

Comment:
```suggestion
                if output.get("body", {}).get("output") is None:
```
You'll need the empty dict default to avoid getting a NoneError